natlas-server/webpack.config.js

```diff
@@ -1,9 +1,21 @@
 /* eslint-env node */
 const path = require('path');
+const webpack = require('webpack');
 const WebpackManifestPlugin = require('webpack-yam-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const assetRootPath = path.resolve(__dirname, 'app', 'static');
 
+function revision() {
+    if (process.env.SOURCE_BRANCH === "main") {
```

Comment:
is `process` just some global or otherwise where is this coming from? Are `SOURCE_BRANCH`, `SOURCE_COMMIT`, `DOCKER_TAG` values that are available in the env by default or does it require extra work for development/deployment?

Reply:
`process` comes from node.js and Webpack. It's a special variable available in those environments.

The environment variables are available in Docker Hub: https://docs.docker.com/docker-hub/builds/advanced/

If you're in a dev environment then these variables would not be available so it uses "dev".